_episodes/02-image-basics.md

```diff
@@ -127,27 +128,27 @@ import ipympl
 > > In the example above, form 1 loads the entire `skimage` library into the
 > > program as an object.
 > > Individual modules of the library are then available within that object,
-> > e.g. to access the `imread` function used in the example above,
-> > you would write `skimage.io.imread()`.
+> > e.g., to access the `disk` function used in the drawing episode,
```

Comment:
Here I should hyperlink to https://datacarpentry.org/image-processing/04-drawing/index.html.

Reply:
The comma is just a convention, but since we find "..., i.e., ..." in the rest of the episode, I think we should have "..., e.g., ..." likewise.

Reply:
```suggestion
> > e.g., to access the `disk` function used in [the drawing episode]({{ page.root }}{% link _episodes/04-drawing.md %}),
```

Reply:
This is a very nice way to adjust the callout to remain relevant while removing the mention of `skimage.io` - thanks @mkcor. 

Above I have suggested the syntax for creating the internal link you mentioned. The internal link syntax for this lesson template is not very intuitive: [one of the many improvements coming with The Carpentries Workbench will be to make internal linking much more straightforward](https://carpentries.github.io/workbench/transition-guide.html#internal-links).

Reply:
> This is a very nice way to adjust the callout to remain relevant

Thanks, @tobyhodges! Plus, I believe it is *plausible* to have an object named `disk` (maybe more likely than `imread`?).